R/check_normality.R

```diff
@@ -68,6 +68,32 @@ check_normality.default <- function(x, ...) {
   p.val
 }
 
+# numeric -------------------
+
+#' @export
+check_normality.numeric <- function(x, ...) {
+  # check for normality of residuals
+  p.val <- .check_normality(x, NULL, type = "raw")
+
+  attr(p.val, "data") <- x
+  attr(p.val, "object_name") <- insight::safe_deparse(substitute(x))
+  attr(p.val, "effects") <- "fixed"
+  class(p.val) <- unique(c("check_normality", "see_check_normality", "check_normality_numeric", class(p.val)))
+
+  p.val
+}
+
+#' @export
+#' @importFrom stats residuals
```

Comment:
If we wanted to not import from stats, we can use `@s3method` instead

Reply:
Compare to how we do pp_check

Reply:
like this?